mtags/src/main/scala/scala/meta/internal/metals/docstrings/ScaladocParser.scala

```diff
@@ -272,7 +272,7 @@ object ScaladocParser {
     for {
       (start, end) <- ScaladocUtils.tagIndex(docstring).iterator
       if ScaladocUtils.startsWithTag(docstring, start, "@define")
-      List("@define", key, value) <- List(
+      case List("@define", key, value) <- List(
```

Comment:
`List(define, key, value) <- .... if define == "@define"` to make it compile?
or `startsWithTag(..., "@define\\s")` and `List(_, key, value)`